cdc/server.go

```diff
@@ -298,12 +298,22 @@ func (s *Server) etcdHealthChecker(ctx context.Context) error {
 func (s *Server) run(ctx context.Context) (err error) {
 	if !config.NewReplicaImpl {
 		kvStorage := util.KVStorageFromCtx(ctx)
+		// if s.capture != nil && s.capture.session != nil {
+		// 	s.capture.session.Close() //nolint:errcheck
+		// }
```

Comment:
Why adding this logic, but commented out

Reply:
sry, it's still testing.